guides/kfp_tekton_install.md

```diff
@@ -76,7 +76,7 @@ To install the standalone Kubeflow Pipelines with Tekton, run the following step
 
 1. Follow the [Kubeflow install instructions](https://www.kubeflow.org/docs/ibm/deploy/install-kubeflow-on-iks/#kubeflow-installation)
    to install the entire Kubeflow stack with `kfp-tekton`.
-   Kubeflow `1.3.0` uses Tekton `v0.21.0` and `kfp-tekton` `v0.8.0`. <!-- TODO update-->
+   Kubeflow `v1.3.1` uses Tekton `v0.21.0` and `kfp-tekton` `v0.8.0`. <!-- TODO update-->
```

Comment:
should it be  `kfp-tekton` `v0.9.0` ?

Reply:
v1.3.1 is still using v0.8.0, otherwise user cannot upgrade from v1.3.0 to v1.3.1.
v0.9.0 (or v0.10.0 if we could make it) will be in kubeflow 1.4 release in september.